sklearn/tests/test_naive_bayes.py

```diff
@@ -261,8 +263,8 @@ def test_discretenb_partial_fit(cls):
         assert_array_equal(clf1.feature_count_, clf3.feature_count_)
 
 
-@pytest.mark.parametrize('cls', [BernoulliNB, MultinomialNB, GaussianNB,
```

Comment:
I am not sure why `GaussianNB` was included in a test of discrete naive Bayes classifiers, but there seemed no harm in leaving it.